shopfloor_dangerous_goods_mobile/README.rst

```diff
@@ -0,0 +1,79 @@
+================================
```

Comment:
I wonder if it would not be better named `shopfloor_mobile_dangerous_goods`

Reply:
that was my first idea, but @simahawk disagrees

Reply:
ok

Reply:
rationale: keep all related modules under the same "namespace". It's also handy when you list directories or browse modules in your preferred editor ;)